spec/controllers/concerns/client_sortable_spec.rb

```diff
@@ -123,20 +134,18 @@
       let(:params) do
         {
             clear: true,
-            search: "query",
-            status: "intake_in_progress",
-            year: "2019",
-            needs_response: true,
-            assigned_to_me: true,
-            unassigned: true,
-            vita_partner_id: 1,
             assigned_user_id: 1
         }
       end
 
-      it "clears all of the existing params" do
+      before do
+        allow(cookies).to receive(:delete)
+        allow(cookies).to receive(:[]).with(anything)
+      end
+
+      it "removes the filter cookie" do
         subject.filtered_and_sorted_clients
-        expect(assigns(:filters).values.compact).to be_empty
+        expect(cookies).to have_received(:delete).with("some_filter_cookie_name")
       end
```

Comment:
Since the `it "removes..."` test validates that we remove the cookie, I'd expect to see a test in this file that verifies we set the cookie, too.